clustered/common/src/main/java/org/ehcache/clustered/common/messages/ServerStoreOpMessage.java

```diff
@@ -99,6 +150,13 @@ public ServerStoreOp operation() {
       this.payload = payload;
     }
 
+    GetAndAppendMessage(ByteBuffer buffer) {
+      super(buffer);
+      byte[] remaining = new byte[buffer.remaining()];
+      buffer.get(remaining);
+      this.payload = ByteBuffer.wrap(remaining).asReadOnlyBuffer();
```

Comment:
Since the buffer gets passed to the super ctor, from line 155 onward I am just reading the rest in the buffer to a byte array and then setting is as payload.